sql/2021/security/cookie_attributes.sql

```diff
@@ -0,0 +1,48 @@
+#standardSQL
+# Cookie attributes (HttpOnly, Secure, SameSite, __Secure- and __Host- prefixes) for cookies set on first-party and third-party requests
+CREATE TEMPORARY FUNCTION getSetCookieHeaders(headers STRING)
+  RETURNS ARRAY<STRING> DETERMINISTIC
```

Comment:
Currently there's a bug in the SQL linter that doesn't like the word `DETERMINISTIC`. Fix has been merged so just waiting for it to be fixed. In the meantime you can set this to be ignore:

```suggestion
-- SQL Linter cannot handle DETERMINISTIC keyword so needs noqa ignore command on previous line
CREATE TEMPORARY FUNCTION getSetCookieHeaders(headers STRING)  -- noqa: PRS
  RETURNS ARRAY<STRING> DETERMINISTIC
```

Same with other lines like this, which are showing this error or similar:

```
L:   3 | P:   1 |  PRS | Line 3, Position 1: Found unparsable section: 'CREATE
96
                       | TEMPORARY FUNCTION getSetCookieHe...'
```